.github/ISSUE_TEMPLATE/request_template.yml

```diff
@@ -0,0 +1,24 @@
+name: Template Request
```

Comment:
Isn't requesting templates better handled through the [CodeExchange process](https://www.twilio.com/code-exchange/submit-code-request)?